--- a/feeds/luci/contrib/package/community-profiles/files/etc/config/profile_potsdam
+++ b/feeds/luci/contrib/package/community-profiles/files/etc/config/profile_potsdam
@@ -1,7 +1,7 @@
 config 'community' 'profile'
 	option 'name' 'Freifunk Potsdam'
 	option 'homepage' 'http://potsdam.freifunk.net'
-	option 'ssid' 'Freifunk-Potsdam-XXX-YYY'
+	option 'ssid' 'freifunk-potsdam.de'
 	option 'mesh_network' '10.22.0.0/16'
 	option 'splash_network' '192.168.22.0/24'
 	option 'splash_prefix' '24'
